### PR TITLE
[BPK-3055] Illustrations page

### PR DESCRIPTION
**Pull request by shaundon**
Won't build because I have excluded illustrations while we find out if we're allowed to. But once I add them back it'll work 💯 

0 0 0 0_8080_guidelines_illustration (2)

### CODE REVIEW

File: docs/src/pages/IllustrationPage/IllustrationPage.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    id: 'intro',
+    content: <BpkMarkdownRenderer source={getMarkdownString(intro)} />,
+  },
+  {
+    id: 'Overview',
+    title: 'Overview',
+    backgroundStyle: 'light',
+    content: (
+      <BpkImage
+        altText="Three examples of illustrations covering a range of situations."
```
Comment (georgegillams) [inline docs/src/pages/IllustrationPage/IllustrationPage.js]:
```suggestion
        altText="Illustration examples."
```

File: docs/src/pages/IllustrationPage/IllustrationPage.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        src={`/${IllustrationOverview}`}
+      />
+    ),
+  },
+  {
+    id: 'examples',
+    title: 'Examples',
+    content: (
+      <BpkImage
+        altText="Five different examples of illustrations covering a range of situations."
```
Comment (georgegillams) [inline docs/src/pages/IllustrationPage/IllustrationPage.js]:
```suggestion
        altText="Illustration examples."
```

File: docs/src/layouts/links.js [hunk 3]
```diff
@@ -527,6 +527,11 @@ export default [
       { id: 'PHOTOGRAPHY', route: routes.PHOTOGRAPHY, children: 'Photography' },
       { id: 'SHADOWS', route: routes.SHADOWS_GUIDELINES, children: 'Shadows' },
       { id: 'COPYWRITING', route: routes.COPYWRITING, children: 'Copywriting' },
+      {
```
Comment (ojcurtis) [inline docs/src/layouts/links.js]: Can this be moved to above copywriting? This is to maintain the new guidelines page order 😄

File: docs/src/pages/IllustrationPage/IllustrationPage.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    content: <BpkMarkdownRenderer source={getMarkdownString(intro)} />,
+  },
+  {
+    id: 'Overview',
+    title: 'Overview',
+    backgroundStyle: 'light',
+    content: (
+      <BpkImage
+        altText="Three examples of illustrations covering a range of situations."
+        width={840}
```
Comment (ojcurtis) [inline docs/src/pages/IllustrationPage/IllustrationPage.js]: Can you double check these heights and widths that are defined in zeplin?

File: docs/src/pages/IllustrationPage/IllustrationPage.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      />
+    ),
+  },
+  {
+    id: 'examples',
+    title: 'Examples',
+    content: (
+      <BpkImage
+        altText="Five different examples of illustrations covering a range of situations."
+        width={840}
```
Comment (ojcurtis) [inline docs/src/pages/IllustrationPage/IllustrationPage.js]: Same as the above

File: docs/src/pages/IllustrationPage/IllustrationPage.js (hunk 4 above)
Comment (shaundon) [reply docs/src/pages/IllustrationPage/IllustrationPage.js]: Addressed 👍

**Pull request merged**